game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -145,7 +145,8 @@ def update_tgo(self, tgo: TheaterGroundObject) -> None:
         if node is None:
             # Not participating
             return
-        # TODO Add the connections or calculate them..
+        if self.advanced_iads and not self.iads_config:
+            self._make_advanced_connections_by_range(node)
```

Comment:
This is what we seem to disagree about. I can understand that you don't wish "fallback" in case the advanced iads was configured manually, but if that's not the case and it was initialized "by range" (which contains subtle mistakes btw) then I don't see why it's considered a problem to recalculate advanced connections.

Worst case I can drop this change, but the mistakes of the "range initialization" do need to be addressed.

Reply:
Sorry i dont get the point of the comment. I see in the code that the node gets recreated but the connections of this node are not yet created. Thats what the todo is about, i get it.
Your change will only recreate the connections when there is no iads_config in the campaign. The recreation with a given config is still missing.

Reply:
If no iads_config is given, we're initializing by range. As I understood what @DanAlbert said about this, nodes are not supposed to reconnect by range in case a config was given, instead they should "degrade" like they currently do. That's why I check for advanced_iads and no config...

Reply:
For anything IADS related, if @RndName and I are telling you different things, @RndName is the one to listen to. I haven't touched this system so unless @RndName except when written down his intent I can only guess at how things are _supposed_ to work.